src/parsers/simplehash.js

```diff
@@ -9,7 +9,12 @@ export const parse = (asset) => {
     // Token descriptions are a waste of space for most collections we deal with
     // so by default we ignore them (this behaviour can be overridden if needed).
     description: asset.description,
+    originalMetadata: JSON.stringify(asset),
     imageUrl: asset.previews?.image_medium_url ?? asset.image_url,
+    imageOriginalUrl: asset.extra_metadata.image_original_url,
+    animationOriginalUrl: asset.extra_metadata.animation_original_url,
+    metadataOriginalUrl: asset.extra_metadata.metadata_original_url,
+    imageProperties: asset.image_properties,
```

Comment:
What is this? Is it only supported by simplehash

Reply:
Are you referring to image_properties? Peter mentioned he wanted to start saving that info: [https://reservoir0x.slack.com/archives/C0582JG5BMM/p1684470237526469](https://reservoir0x.slack.com/archives/C0582JG5BMM/p1684470237526469). It just has some basic info about the image type, width and height